matrix-meetings-bot/src/service/MeetingService.ts

```diff
@@ -606,6 +606,14 @@ export class MeetingService {
       }
 
       promises.push(this.cleanupWidgets(widgetIds, room, userContext));
+      const filteredWidgetIds = widgetIds.filter(
+        (widget) =>
+          !widget.startsWith(`${WidgetType.COCKPIT}`) &&
+          !widget.startsWith(`${WidgetType.BREAKOUT_SESSIONS}`)
```

Comment:
There is no need to use template string here.

Reply:
you are absolutely right!